### ISSUE TITLE
Decentralized Hub for News Fact-Checking

### ISSUE THREAD

**Issue by KianaH** [Blockternship/projects#8]
Hello everyone, my name is Kiana! I'm a computer science student with an interest in blockchain technology and smart contracts. I'm also interested in how we can harness the open source community in educating others.

## Overview
Fake News is running rampant. Everyday we are bombarded with dubious claims and questionable statistics. With the emergence of blockchain technologies, we have the power to work as a community to verify the authenticity of the news we read, hear, and watch.

My goal is to create an open source dapp where users can pose questions (and attach references for those questions) based on online news to the community and have verified users determine the veracity of that news and provide sources to support their answers. We would use a blockchain identity solution to ensure that the community is not trolled. The use of a person’s real identity incentivizes them to give truthful answers and provide valid sources for their claims. Other users will also be able to determine whether the answers given are accurate, dispute answers, and give follow-ups to answers.

Each community member that actively participates will also have reputation within the platform and the more they accurately answer questions and provide accurate sources, the higher their reputation will be.

## Philosophy
I have an interest in information and how it’s transmitted to the public. However, I’ve found that with the rise of social media, it has become profitable for fake news to flourish. I want to curb the spread of fake news by allowing journalists and other experts in their field to correct inaccuracies and bring integrity back to journalism.

## Next Steps
I have experience with web development and Ethereum dapps, but I'm currently looking for teammates with experience with UI/UX and/or web development as well.

I believe this project would most align with @Giveth because the emphasis on community collaboration and social altruism is also a major goal that I have with this project.

**Contact Info**
Github: @KianaH
Keybase: @kianaharrell

**Comment by arminvoid** [comment]
(I do not belong to the blockinternship organizers team)

Sounds similar to what [Augur](https://www.augur.net/) is trying to do. Maybe you can buidl this specific use case on top of Augur.
Also, you can gamify the process of incentivising agents to give truthful answers by making them deposit a collateral and penalising them for wrong answers.

You wrote about _verifying_ the users. I am guessing the agency that verifies these users is intended to be centralized and that defeats the spirit of the blockchain.

**Comment by YalorMewn** [comment]
Hey @atvanguard you don't need to belong to the organizers to comment on these issues, we welcome your thoughts ! In response to your question about centralization for verifying users, there's a solution that happens to be in development in the Social Coding Circle called [BrightID](
https://github.com/brightID) which uses a trusted group of close personal contacts plus a decentralized graph database to verify your unique person good. 

@kianaH this is definitely something I think could bring a lot of value and credibility to the space if done right. Would love to see you connect with @griffgreen and give some thought to EIP0 at some point also.

**Comment by KianaH** [comment]
@atvanguard Thank you for that gamification suggestion! And ideally if BrightID is released during the hackathon, I would use it as a solution for the verification issue. Given that it's still in development at the moment, if I am not able to use that as a solution, my tentative approach is something similar to how you provide proof of identity on keybase or how you'd use certain Ethereum testnet faucets by posting your address to social media. Thus, other users could verify your online identity for themselves without a strict centralized verifier. 

I'd also consider putting into place measures where users could dispute a form of proof given by another user, but with the financial incentives in place, that should incentivize people to provide valid proof. However, I do eventually want to utilize a form of verification that allows for anonymity while giving users the assurance that the person is verified (which can be done with BrightID as far as I know).

**Comment by kikipluche** [comment]
i don't see why it is so important to have a verified profile. The goal is to have validated news. anyone (verified or not) can verify news. when others validate that, the person gets reputation (or bad rep when fake news is validated), for me the most important thing will be the reputation part, not what "real" identity lies behind that. In fact, that is of no importance.

**Comment by deamme** [comment]
@kikipluche The challenge is how you prevent spammers/bots from upvoting content that they are benefiting from where verified identity is one of the solutions.

**Comment by KianaH** [comment]
@kikipluche Thanks for the feedback. Verification is important because a verified profile is a deterrent for making multiple accounts and puts more at stake (a person's reputation) when a user answers a question. Online communities are oftentimes plagued by multiple accounts per user being used to sway opinions or give an illusion of consensus. By having verified users, we could enforce a one-person-one-account rule (ideally with the BrightID solution or a similar mechanism) while also being able to verify that the person answering the question has some knowledge of the subject matter (when necessary). While there are cases where anonymity is pertinent for the user, for most cases, being able to verify a person's identity and know who is providing you sources is paramount.

While it's easy to verify whether someone posted a certain tweet or a politician said a certain quote, it's a lot more difficult to verify specialized knowledge that may require more background to even understand. This is especially the case with science-related news that is oftentimes exaggerated or fabricated for increased viewership. In this case, it's important for me to know the credentials of the individual answering the question because even though two people may cite valid sources, someone that has studied that field would have a better understanding of the sources and would be less likely to misinterpret or misuse the findings in the source.

**Comment by kikipluche** [comment]
@KianaH ok, i get it, so on the one side, you want people to bring in their reputation from outside your tool. And on the other side, you want to prevent some sort of "fraud" by someone creating a lot of profiles and verifying him/herselve. The first one is a choice you make "yes i find it important that people can bring in reputation from outside" (which you have no control over, imo), the second one relates to spam control. You can look at different ways to prevent someone to do this, fe having to stake tokens, or spend tokens to review. So it is economic not interesting to be validating yourself with different identities. Just thinking out loud here :-)

**Comment by kikipluche** [comment]
@deamme exactly, verified identity is one of the solutions. Looks like the most obvious one, but it is the one i like the least ;-), (just from my super personal point of view, that is).

**Comment by KianaH** [comment]
@kikipluche, I appreciate the suggestion of a token system. However, a staking economy won't prevent wealthy individuals and high net-worth corporations from influencing & reaffirming "fake news." Verification adds accountability.

**Comment by YalorMewn** [comment]
Hey @KianaH some great thoughts happening here in the comments ;) 

Your project is a perfect fit for the #Blockternship. So welcome, can we start a new repo for you or is there one we can fork in ?

**Comment by KianaH** [comment]
Hi @yalormewn, could you start a new repo? Thank you :)

**Comment by YalorMewn** [comment]
Here you https://github.com/Blockternship/Issue_9 you have full admin access, name it as you see fit 👍

**Comment by dmitry-christie** [comment]
Hey @KianaH - I'm a frontend web dev. Your project sounds interesting. Are you still looking for teammates?

**Comment by KianaH** [comment]
Hi @dmitry-christie! Yes I am actually. If possible could you message me on keybase?

**Comment by YalorMewn** [comment]
Hey @KianaH I would like to invite you to the first call of the hackathon, would you send me a quick email here YalorTackson@gmail.com also you can add yourself to the event [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=NjlxM2dtdDd2bHA5ODNxbzNsYm5vOTMyZDkgeWFsb3J0YWNrc29uQG0&tmsrc=yalortackson%40gmail.com)

**Comment by adamstallard** [comment]
>And ideally if BrightID is released during the hackathon, I would use it as a solution for the verification issue.

>I do eventually want to utilize a form of verification that allows for anonymity while giving users the assurance that the person is verified (which can be done with BrightID as far as I know).

I can't wait for the day when I can just point people to BrightID and say "this is exactly what you need; use this." We're launching our beta in November.

Yes, it does allow for anonymity, which I think will be a strong point for your project.

To me, the really unique, valuable part of your project is being able to limit each person to one account, doing away with all the manipulation that can be done with bot nets.

**Comment by status-open-bounty** [comment]
Current balance: 0.0 ETH
Contract address: [0xe80cc972b878a742ef25db3403f0fdfd3ecd05db](https://etherscan.io/address/0xe80cc972b878a742ef25db3403f0fdfd3ecd05db)
![QR Code](https://openbounty.status.im/qr/Blockternship/projects/bounty/8/a53c67874ef84fa6dc519757f7f70c9dd66a405f633671071973a6cf37a0336c/qr.png)
Network: Mainnet
To claim this bounty sign up at [https://openbounty.status.im](https://openbounty.status.im) and make sure to update your Ethereum address in `My Payment Details` so that the bounty is correctly allocated.
To fund it, send ETH or ERC20/ERC223 tokens to the contract address.

**Comment by YalorMewn** [comment]
New issue #22
